src/notification/service/notificationService.ts

```diff
@@ -129,28 +132,27 @@ class NotificationService {
                     setting.event_type_id = event.eventTypeId
                     setting.pipeline_id = 0
                     setting.config = event.payload
-                    h.sendAndLogNotification(event, sdk,setting,{"dest": "slack"}, slackTemplateConfig)
-                })
+                    await h.sendAndLogNotification(event, sdk,setting,{"dest": "slack"}, slackTemplateConfig)
```

Comment:
We need to test if await is required everywhere as this might increase the response time